Marlin/src/gcode/calibrate/G28.cpp

```diff
@@ -440,4 +444,12 @@ void GcodeSuite::G28(const bool always_home_all) {
   #if ENABLED(DEBUG_LEVELING_FEATURE)
     if (DEBUGGING(LEVELING)) SERIAL_ECHOLNPGM("<<< G28");
   #endif
+
+  #if HAS_DRIVER(L6470)
+    // Set L6470 absolute position registers to counts
+    for (uint8_t j = 1; j <= L6470_chain[0]; j++) {
```

Comment:
So, I see a lot of references to `L6470_chain` but I don't see a definition for it anyplace.